### ISSUE TITLE
Logo is missing in modlist :wacko:

### ISSUE THREAD

**Issue by Alwinfy** [williewillus/botania-fabric-issues#42]
![image](https://user-images.githubusercontent.com/20421383/103324147-bc595980-4a0b-11eb-9e3e-157826ab1af7.png)

**Comment by quat1024** [comment]
This is because ModMenu explodes on non-square pics, this gets dumped in the log:
```
[00:40:09] [main/ERROR]: Invalid icon for mod Botania
java.lang.IllegalStateException: Must be square icon
	at org.apache.commons.lang3.Validate.validState(Validate.java:829) ~[commons-lang3-3.5.jar:3.5]
	at io.github.prospector.modmenu.gui.ModListEntry.createIcon(ModListEntry.java:97) [modmenu-1.14.13+build.19.jar:?]
```

**Comment by williewillus** [comment]
idk how to make the current logo file look nice in a square

**Comment by TheEssem** [comment]
> idk how to make the current logo file look nice in a square

Might I suggest cropping it to the "o" in the current Botania logo? It seems stylized enough to stand out and still be recognizable as Botania.

**Comment by williewillus** [comment]
fixed by Vazkii/Botania#3618
